.github/workflows/ci.yml

```diff
@@ -50,6 +50,10 @@ jobs:
 
       - name: Load cache
         uses: Swatinem/rust-cache@v2
+        with:
+          # use a separate cache for each job to work around
+          # https://github.com/Swatinem/rust-cache/issues/124
+          key: "${{ matrix.name }}"
```

Comment:
I wonder if this might lead us to constantly overflow the cache capacity - see warning at https://github.com/matrix-org/matrix-rust-sdk/actions/caches.

Reply:
yeah, I'll probably limit persisting the cache to the main branch.